services/catalog/src/simcore_service_catalog/core/background_tasks.py

```diff
@@ -29,12 +29,14 @@
 from ..db.repositories.services import ServicesRepository
 from ..services import access_rights
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
 # NOTE: by PC I tried to unify with models_library.services but there are other inconsistencies so I leave if for another time!
-ServiceKey = str
-ServiceVersion = str
-ServiceDockerDataMap = dict[tuple[ServiceKey, ServiceVersion], ServiceDockerData]
+ServiceKey: TypeAlias = str
```

Comment:
we still need to discuss ```NewType``` vs ```TypeAlias```

Reply:
Ah I see your point. `NewType("ServiceKey", str)` would be something like
```python
class NewType(str):
     ...
```

And therefore creating a new type as a subclass of `str`.